### ISSUE TITLE
Text re-sizing for increaased usability scenarios

### ISSUE THREAD

**Issue by SteadfastSeb** [gbevin/ShowMIDI#22]
I ran into the problem of readability with my multi monitor setup, the text became too small to discern on the furthest screen, which is the best place for it to be for my workflow.

Text options would help with odd scenarios like mine and folks with minor vision impairments.

After loading up the program and checking again, I think at least a 4X size increase would be needed for my use case to work well.

The font itself is less of an issue, it's sheer size that is the problem. The method to achieve the scaling is not a concern, UI scaling would be fine instead of arbitrary text loading.

Maybe 4 clearly distinct sizes, like, 1X, 2X, 4X, 8X perhaps if things need to remain in perfect scaling.

Overall, love that you made the tool to begin with, really helps with troubleshooting my odd setup 👍

**Comment by RadekPilich** [comment]
Yes, I've resorted to using Windows magnifier when analyzing / troubleshooting some device MIDI and not sitting right in front of the computer screen.

**Comment by peternicholls** [comment]
I've forked this project for further development. I have monitor DPI aware scaling working on the main branch. i've not built a windows .exe for this however, I'm having to use github workflows to compile and build for windows for the time being. But there is mac and linux version too. If you'd like to check those out, I'm actively maintaining the fork repo. In due course I'll cherry pick and submit a pull request to Gavin, if he wishes to receive that

[ShowMIDI-fork v 1.1.3](https://github.com/peternicholls/ShowMIDI-Fork/tree/main)
